lib/cisco_node_utils/client/nxapi/client.rb

```diff
@@ -286,6 +286,14 @@ def handle_output(prev_cmds, command, output)
       # handle accordingly
       fail Cisco::RequestNotSupported, \
            "Structured output not supported for #{command}"
+    # Error 432: Requested object does not exist
+    # Ignore 432 errors because it means that a property is not configured
+    elsif output['code'] =~ /4\d\d/ && output['code'] != '432'
+      fail Cisco::RequestFailed, \
+           "#{output['code']} Error: #{output['msg']}"
+    elsif output['code'] =~ /5\d\d/
```

Comment:
Seems like you could combine these two cases to avoid duplicating the 'fail' entry:

```ruby
elsif output['code'] =~ /[45]\d\d/ && output['code'] != '432'
```